ecosystem/indexer-grpc/indexer-grpc-data-service/src/metrics.rs

```diff
@@ -116,3 +116,30 @@ pub static BYTES_READY_TO_TRANSFER_FROM_SERVER: Lazy<IntCounterVec> = Lazy::new(
     )
     .unwrap()
 });
+
+/// Size of the data service grpc stream sender channel.
+pub static GRPC_STREAM_RESPONSE_CHANNEL_SIZE: Lazy<IntGaugeVec> = Lazy::new(|| {
+    register_int_gauge_vec!(
+        "data_service_grpc_stream_thread_channel_size",
+        "Size of the data service grpc stream sender thread channel",
+        &[
+            "start_version",
+            "connection_id",
```

Comment:
These fields are for labeling the metric. They're used to differentiate what is being measured

Since we want to measure the channel size per connection, I don't think we need `start_version` here. `connection_id` makes sense, but we should try to keep it consistent with the other metrics which use
```
            "identifier_type",
            "identifier",
            "email",
            "application_name",
            "processor"
```

Reply:
If we want a metric that measures the latest version in the channel per processor, that should be a separate metric.